moderngl_window/atlas/simple_atlas.py

```diff
@@ -116,13 +115,13 @@ def __init__(
         self._auto_resize = auto_resize
 
         # The physical size limit for the current hardware
-        self._max_size = self._ctx.info["GL_MAX_VIEWPORT_DIMS"]
+        self._max_size: tuple[int, int] = self._ctx.info["GL_MAX_VIEWPORT_DIMS"]
 
         # Atlas content
         self._texture = self._ctx.texture(self.size, components=self._components)
 
         # We want to be able to render into the atlas texture
-        self._fbo = self._fbo = self._ctx.framebuffer(color_attachments=[self._texture])
```

Comment:
There is no reason there should be `self._fbo = self._fbo = ...` apart from a mistake

Reply:
yup :)